### DISCUSSION TITLE
Issue while deploying

### DISCUSSION THREAD

**Discussion by lennycode** [imixs/imixs-jsf-example#64] [General]
Using the Docker domain.xml with Payara.

I made some changes to get PGSQL to work.

**I get this error when I deployed:**

`......
Caused by: java.nio.file.InvalidPathException: Trailing char < > at index 18: imixs-sample-index `

**Then when I deploy again, I get;**
(The tables were obviously created, a good thing!0

`Error occurred during deployment: Exception while loading the app : javax.ejb.CreateException: Initialization failed for Singleton SetupService. Please see server.log for more details. PER01003: Deployment encountered SQL Exceptions: PER01000: Got SQLException executing statement "CREATE TABLE EVENTLOG (ID VARCHAR(255) NOT NULL, CREATED TIMESTAMP, DATA BYTEA, REF VARCHAR(255), TOPIC VARCHAR(255), VERSION INTEGER, PRIMARY KEY (ID))": org.postgresql.util.PSQLException: ERROR: relation "eventlog" already exists PER01000: Got SQLException executing statement "CREATE TABLE DOCUMENT (ID VARCHAR(255) NOT NULL, CREATED TIMESTAMP, DATA BYTEA, MODIFIED TIMESTAMP, TYPE VARCHAR(255), VERSION INTEGER, PRIMARY KEY (ID))": org.postgresql.util.PSQLException: ERROR: relation "document" already exists PER01003: Deployment encountered SQL Exceptions: PER01000: Got SQLException executing statement "CREATE TABLE EVENTLOG (ID VARCHAR(255) NOT NULL, CREATED TIMESTAMP, DATA BYTEA, REF VARCHAR(255), TOPIC .... msg.seeServerLog`

**Comment by rsoika** [comment]
Can you check the file 'imixs.properties' if there is a trailing ' ' char at the end of the index directory. Try to remove this . Maybe this can cause the problem

**Comment by lennycode** [comment]
I guess this:

![image](https://user-images.githubusercontent.com/845855/67620926-f0b70180-f7d9-11e9-93fd-415e70c8b363.png)

**Comment by rsoika** [comment]
yes - does it work if your remove the blank and rebuild your application again?

**Comment by lennycode** [comment]
works!

**Comment by rsoika** [comment]
thanks for this hint. We will fix this in next release.
